tools/build_defs/cc_import.bzl

```diff
@@ -0,0 +1,208 @@
+"""Modified cc_import script; to be merged with cc_import?
```

Comment:
This seems to not define `cc_import`. Please document what this file is about :)